libnavui-dropin/build.gradle

```diff
@@ -45,13 +45,12 @@ android {
 dependencies {
     api project(":libnavui-base")
     api project(":libnavui-maneuver")
-    api project(":libnavui-maps")
     api project(":libnavui-speedlimit")
     api project(":libnavui-tripprogress")
     api project(":libnavui-voice")
+    api project(":libnavui-app")
```

Comment:
We should revert this change as the idea is to not expose `libnavui-app` transitively just as we do with `libnavui-utils` and `libnavui-resources`.

Reply:
Yeah, i was going back and forth with the location for `NavigationApi`. If it exists in `:libnavui-app` then this would be exposed transitively.

`NavigationViewApi` would not be shared with android auto, and can develop opinions specific to `NavigationView`.

Changed this back to `implementation`